### ISSUE TITLE
Console, alert and footer option not working.

### ISSUE THREAD

**Issue by csalvato** [flyerhzm/bullet#513]
Currently developing a Rails app and we are using React Router to handle routing (mentioning it because I don't know if that would impact these options).

With this configuration in my `development.rb`, I am seeing Bullet warnings in my rails console, but not in a footer, console or alert.

```ruby
  config.after_initialize do
    Bullet.enable = true
    Bullet.alert = true
    Bullet.console = true
    Bullet.add_footer = true
    Bullet.rails_logger = true
  end
```

Is there something I should be doing that I'm not? 🤔 

If more information is necessary please let me know.

**Comment by flyerhzm** [comment]
@csalvato could you give version 6.1.2 a try?

**Comment by csalvato** [comment]
Thanks @flyerhzm !

I upgraded to 6.1.2, and added this to my `development.rb` in Rails:

```rb
config.after_initialize do
    Bullet.enable = true
    Bullet.alert = true
    Bullet.console = true
    Bullet.add_footer = true
    Bullet.rails_logger = true
end
```

It does not appear to have had any impact.

**Comment by vpiau** [comment]
Same here with version `7.0.1` on Rails 5.2 project.

I tried on several browsers and empty caches.

**Comment by flyerhzm** [comment]
it's possible if response body does not contain `html` tag, I have fixed in version `7.0.2`
